src/routes/api/users.js

```diff
@@ -74,9 +73,20 @@ router.post('/users', (req, res, next) => {
   user.email = req.body.user.email;
   user.setPassword(req.body.user.password);
 
-  user.save()
+  user
+    .save()
     .then(() => res.json({ user: user.toAuthJSON() }))
     .catch(next);
 });
 
+// @route POST /api/v1/users/forgotpassword
+// @desc Generate User Password Reset / Returning JWT Token
+// @access Public
+router.post('/forgotpassword', forgotPassword);
```

Comment:
There should be middlewares to validate users authentications.

Reply:
The idea was to allow anyone to request password reset so as not to expose  application membership to non users. The implementation was designed to send sign up mail instead of reset mail to non users. Is that good enough for a consideration?

Reply:
OK good. Also, please version your Apis, reach out to @zanio